scss/titles.scss

```diff
@@ -0,0 +1,19 @@
+.title {
+  color: #000;  
+  font-size: 1.4rem;
+  height: 90px;
+  margin: 35px 70px;
```

Comment:
remove this margins for now, lets just keep the styles related to the component and not to where the component is placed.